usaspending_api/awards/v2/filters/location_filter_geocode.py

```diff
@@ -1,8 +1,14 @@
+from django.conf import settings
 from django.db.models import Q
+from itertools import chain
+from typing import Optional
+
+from usaspending_api.common.elasticsearch.client import es_client_query
 from usaspending_api.common.exceptions import InvalidParameterException
+from usaspending_api.search.v2.elasticsearch_helper import es_sanitize
 
 
-def geocode_filter_locations(scope, values, use_matview=False):
+def geocode_filter_locations(scope: str, values: list, use_matview: bool = False) -> Q:
```

Comment:
What is `-> Q`, and why are we strongly typing just these params?

Reply:
These aren't strongly typed, these are [type hints](https://docs.python.org/3.5/library/typing.html)

[Q](https://docs.djangoproject.com/en/1.11/ref/models/querysets/#django.db.models.Q)